go.mod

```diff
@@ -1,16 +1,17 @@
 module github.com/lightninglabs/neutrino
 
 require (
-	github.com/btcsuite/btcd v0.22.0-beta.0.20220207191057-4dc4ff7963b4
-	github.com/btcsuite/btcd/btcec/v2 v2.1.0
-	github.com/btcsuite/btcd/btcutil v1.1.0
+	github.com/btcsuite/btcd v0.22.0-beta.0.20220413172512-bf64c8bdbbbf
```

Comment:
CI looks to be failing, I think we need to bump this further? 

Or we need to update everything to target go 1.18 as btcd is using _only_ those new directives?